### ISSUE TITLE
error on file conversion

### ISSUE THREAD

**Issue by Johnfarcy** [dobrosketchkun/bionic-reading-epub-converter#6]
“This page contains the following errors:error on line 1 at column 9: Extra content at the end of the document
Below is a rendering of the page up to the first error.”

**Comment by dobrosketchkun** [comment]
Yeah, I don't really know the epub format specs particularly well (more like at all), so... 
That's why I used the word “rudimentary” in the description.
